src/DependencyInjection/SonataNewsExtension.php

```diff
@@ -141,6 +141,12 @@ public function registerDoctrineMapping(array $config)
 
         foreach ($config['class'] as $type => $class) {
             if (!class_exists($class)) {
+                /*
+                 * NEXT_MAJOR:
+                 * Throw an exception if the class is not defined
+                 */
+                @trigger_error('The class '.$class.' is not defined or doesn\'t exist. This is tolerated now but will be forbidden in 4.0', E_USER_DEPRECATED);
```

Comment:
What is the difference between not being defined and not existing?

Reply:
Forget to declare 
```            
class:
        tag:        App\Application\Sonata\ClassificationBundle\Entity\Tag
```
is not the same as declare a class with a typo in the namespace. 
Maybe it will be easier for the developer to understand his/her mistake if we write these 2 possibilities.

Reply:
You mean writing the name of a class in the file, but forget to create the file with the class in it? Isn't that a bit unlikely?

Reply:
For example : If you copy / paste the advanced configuration, but you don't run sonata extends on classification bundle. You 'll have:
```
class:
        tag:        Application\Sonata\ClassificationBundle\Entity\Tag
```
->  the tag class will be 'defined', but it doesn't really exist.

Reply:
Oh oh that's a bit more likely indeed 👍